src/nnet3/nnet-simple-component.cc

```diff
@@ -154,6 +186,8 @@ void DropoutComponent::Read(std::istream &is, bool binary) {
   ReadBasicType(is, binary, &dim_);
   ExpectToken(is, binary, "<DropoutProportion>");
   ReadBasicType(is, binary, &dropout_proportion_);
+  ExpectToken(is, binary, "<DropoutPerFrame>");
```

Comment:
@GaofengCheng, you need to understand what Vimal was saying here- there needs to be back compatibility code for the old format.  Search for ReadToken() in the file for examples.

However, the reason for your error is that you need to recompile in 'chainbin/' (and possibly chain/').